loader/loader.go

```diff
@@ -182,12 +199,16 @@ func (p *Program) Parse() error {
 
 	// Parse all packages.
 	for _, pkg := range p.Sorted() {
-		err := pkg.Parse()
+		err := pkg.Parse(includeTests)
 		if err != nil {
 			return err
 		}
 	}
 
+	if compileTestBinary {
+		p.SwapTestMain()
```

Comment:
It looks like this returns an error that is not handled: `error: invalid source`.